institution/views.py

```diff
@@ -131,7 +131,7 @@ def form_factory(self) -> Callable[..., ModelForm]:
 
 @login_required
 def full_form(request, pk):
-    inst = get_object_or_404(CofkUnionInstitution, pk=pk)
+    inst: CofkUnionInstitution | None = get_object_or_404(CofkUnionInstitution, pk=pk)
```

Comment:
typing should be 
`inst: CofkUnionInstitution = get_object_or_404(CofkUnionInstitution, pk=pk)`
because `get_object_or_404` never return None